src/SerilogWeb.Classic/Classic/SerilogWebClassicConfigurationBuilder.cs

```diff
@@ -113,6 +116,18 @@ public SerilogWebClassicConfigurationBuilder LogAtLevel(LogEventLevel level)
             return this;
         }
 
+        /// <summary>
+        /// Configure at which level HTTP requests are logged.
+        /// Default is Information
```

Comment:
maybe change that a bit ? 

```suggestion
        /// Configure at which level HTTP requests are logged dynamically depending on the context
```